packages/matrix-identity-server/src/userdb/ldap.ts

```
@@ -14,6 +14,17 @@ class UserDBLDAP implements UserDBBackend {
 
   constructor(conf: Config, private readonly logger: TwakeLogger) {
     this.logger = logger
+
+    if (conf.ldap_uri == null || conf.ldap_uri.length === 0) {
```

Biome (2.4.9)

[error] Using == may be unsafe if you are relying on type coercion.

(lint/suspicious/noDoubleEquals)